zeppelin-web/src/app/notebook/paragraph/paragraph-control.html

```diff
@@ -26,6 +26,20 @@
   <span class="icon-control-play" style="cursor:pointer;color:#3071A9" tooltip-placement="top" tooltip="Run this paragraph (Shift+Enter)"
         ng-click="runParagraph(getEditorValue())"
         ng-show="paragraph.status!='RUNNING' && paragraph.status!='PENDING' && paragraph.config.enabled"></span>
+  <span ng-show="paragraph.runtimeInfos.jobUrl.length == 1">
+    <a href="{{paragraph.runtimeInfos.jobUrl[0]}}" target="_blank"><span class="fa fa-tasks"></span> Spark job </a>
```

Comment:
@1ambda Clearing the links when we clear the output 👍 
@AhyoungRyu Fixed font issue
@Leemoonsoo Added `ParagraphRuntimeInfos` class(https://github.com/apache/zeppelin/pull/1663/files#diff-a2ac97dfb3f07b2a3fd1533af701295e) to have a generic support to add more properties. Let me know how it looks.
@AhyoungRyu @Leemoonsoo @1ambda Fixed the alignment and capitalized the text(to access the jobs) 
![spark](https://cloud.githubusercontent.com/assets/5082742/20637771/4efa0f4a-b3b7-11e6-8845-c4ea819f4aa4.gif)